soc/xtensa/intel_s1000/dts_fixup.h

```diff
@@ -46,4 +46,17 @@
 
 #define CONFIG_I2C_0_IRQ_FLAGS			SNPS_DESIGNWARE_I2C_80400_IRQ_0_SENSE
 #define CONFIG_I2C_0_IRQ_PRI			SNPS_DESIGNWARE_I2C_80400_IRQ_0_PRIORITY
+
+
+#define CONFIG_SPI_0_BASE_ADDRESS		SNPS_DESIGNWARE_SPI_E000_BASE_ADDRESS
+#define CONFIG_SPI_0_NAME			SNPS_DESIGNWARE_SPI_E000_LABEL
+
+#define CONFIG_SPI_0_IRQ			((SNPS_DESIGNWARE_SPI_E000_IRQ_0 << 16) | \
```

Comment:
why this complex macro? Can't you just set the right irq number directly in the .dtsi file?

Reply:
No it is not possible to get single interrupt number from .dtsi.